deps/rabbitmq_peer_discovery_aws/test/integration_SUITE.erl

```diff
@@ -38,153 +42,121 @@ groups() ->
     ].
 
 init_per_suite(Config) ->
+    inets:start(),
     rabbit_ct_helpers:log_environment(),
     Config1 = rabbit_ct_helpers:set_config(
-                Config,
-                [
-                 {terraform_files_suffix, rabbit_ct_helpers:random_term_checksum()},
-                 {terraform_aws_ec2_region, "eu-west-1"},
-                 {rmq_nodes_clustered, false}
-                ]),
-    Config2 = init_aws_credentials(Config1),
-    rabbit_ct_helpers:run_setup_steps(Config2).
+                Config, [
+                         {ecs_region, "eu-west-1"},
+                         {ecs_cluster_name, "rabbitmq-peer-discovery-aws"},
+                         {ecs_profile_name, "rabbitmq-peer-discovery-aws-profile"},
+                         {ecs_instance_role, "ecs-peer-discovery-aws"},
+                         {ecs_cluster_size, ?CLUSTER_SIZE},
+                         {rabbitmq_default_user, "test"},
+                         {rabbitmq_default_pass, rabbit_ct_helpers:random_term_checksum()},
+                         {rabbitmq_erlang_cookie, rabbit_ct_helpers:random_term_checksum()}
+                        ]),
+    Config2 = rabbit_ct_helpers:register_teardown_step(Config1, fun aws_ecs_util:destroy_ecs_cluster/1),
+    rabbit_ct_helpers:run_steps(
+      Config2, [
+                fun rabbit_ct_helpers:init_skip_as_error_flag/1,
```

Comment:
This function wasn't exported, I pushed a `rabbitmq-ct-helpers` change directly

Reply:
Yes, I made that change locally and forgot to push it. Thanks.